$OpenBSD: patch-src_networking_c,v 1.8 2021/03/17 06:25:55 tb Exp $

time_t is long long, so use llabs to avoid truncation
https://github.com/redis/redis/pull/8662

Index: src/networking.c
--- src/networking.c.orig
+++ src/networking.c
@@ -3007,7 +3007,7 @@ void securityWarningCommand(client *c) {
     static time_t logged_time;
     time_t now = time(NULL);
 
-    if (labs(now-logged_time) > 60) {
+    if (llabs(now-logged_time) > 60) {
         serverLog(LL_WARNING,"Possible SECURITY ATTACK detected. It looks like somebody is sending POST or Host: commands to Redis. This is likely due to an attacker attempting to use Cross Protocol Scripting to compromise your Redis instance. Connection aborted.");
         logged_time = now;
     }
